packet/routes/shared.py

```diff
@@ -16,6 +16,7 @@ def freshman_packet(uid, info=None):
     signatures = get_signatures(uid)
     required = sum(get_number_required(uid).values())
     signed = sum(get_number_signed(uid).values())
+    signed_dict = get_number_signed(uid)
```

Comment:
This is likely duplicating a query or similar time consuming operation unnecessarily.